### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by TomBZombie** [ashkitten/g933-utils#8]
Do you have any examples of using this? In particular I'd like to set the 7.1 profile on, I tried doing this in windows but Linux still only recognises it as a stereo device.

**Comment by ashkitten** [comment]
that setting is just a trick used by the driver to simulate surround sound. the setting is not even passed to the headset. you cannot have real 7.1 audio with only 2 drivers; that would be impossible, so the windows driver simulates this using some [HRTF](https://en.wikipedia.org/wiki/Head-related_transfer_function) trickery. i'm sure there are similar plugins for pulseaudio or whatever you use.

**Comment by TomBZombie** [comment]
Thanks, I'd thought the device would show as 7.1 in Linux as it does in Windows with the setting enabled so that applications can send audio to the correct (virtual) speaker.

**Comment by ashkitten** [comment]
yeah, no. it does all of that processing on the computer's end, not the headset.
